Makefile

```diff
@@ -63,6 +63,13 @@ statusgo-ios-simulator: xgo	##@cross-compile Build status-go for iOS Simulator
 	$(GOPATH)/bin/xgo --image farazdagi/xgo-ios-simulator --go=$(GO) -out statusgo --dest=$(GOBIN) --targets=ios-9.3/framework -v $(shell build/testnet-flags.sh) ./lib
 	@echo "iOS framework cross compilation done."
 
+statusgo-library: ##@cross-compile Build status-go as static library for current platform
+	@echo "Building static library..."
+	@go build -buildmode=c-archive -o $(GOBIN)/libstatus ./lib
```

Comment:
I think we may want to see the output of this line so no need for `@` prefix.

Also, I used this:
```
CGO_FLAGS=-mmacosx-version-min=10.7 CGO_LDFLAGS=-mmacosx-version-min=10.7 go build -installsuffix status -ldflags -v -buildmode=c-archive -o libstatus.a ./lib
```
to prevent from displaying warnings when building node.js module.

Reply:
@adambabik what's those `CGO-` flags for? I've built without them and it seems to work fine with node-gyp, but I can double check if needed.

Reply:
It tells CGO which macOS SDK to use. In `node-gyp`, we use `"MACOSX_DEPLOYMENT_TARGET": "10.7"` so it should match with the SDK used to create `libstatus.a`.

Reply:
Btw. there is a typo, it should be `-o $(GOBIN)/libstatus.a`.

Reply:
@adambabik I tried building with those flags and the only difference I can observe on my machine (latest High Sierra) are more warnings about versions mismatch. `(ld: warning: object file (/bin/libstatus.a(000000.o)) was built for newer OSX version (10.13) than being linked (10.7))`